Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "Nemo"
 uuid = "2edaba10-b0f1-5616-af89-8c11ac63239a"
-version = "0.27.3"
+version = "0.28.0-DEV"
```

Comment:
I thought this is a good idea because it's technically a breaking change, but of course this breaks the OscarCI tests against Oscar in this repo, so perhaps I should drop this change from the PR? @thofma @wbhart?

Reply:
Yes, please drop it.

Reply:
And it doesn't look like a breaking change, people on older Julia versions will just not see new Nemo releases.